db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
I didn't think about it first, but the sensors can also be created in the backend by a reporter. The new sensor form would then need a field for the animal_id and a validation that there can only be one animal_id together with a specific sensor type (e.g. only one temperature sensor for a specific animal).